src/core/schema/config.model.schema

```diff
@@ -291,6 +291,19 @@
     },
     "themeSettings": {
       "type": "object"
+    },
+    "_inview": {
+      "type": "object",
+      "title": "jquery.onscreen (inview)",
+      "properties": {
+      "_allowScrollOver": {
+        "type": "boolean",
+        "default": true,
```

Comment:
This raises an interesting question... if I understand this correctly, `_allowScrollOver: true` was really only needed for 'browsers that pause script execution on scroll' i.e. Safari for iOS 6 & 7, neither of which are supported any more (by Adapt or by Apple).

So, do we continue to default it to `true` (to maintain backwards compatibility with existing courses) or change it to `false` because that's the better setting for the current releases of iOS?

(From what I can see it defaults to `true` in the code if there's no setting present in config.json)

Reply:
@moloko good question; although the legacy branch is designed to support IE8-10, what is the support policy for iOS and Android?

Reply:
for iOS it's that we support latest version and one version back (so currently 11 & 12). for Android we're considerably more generous and say that it's any version capable of running Chrome.

If no-one has a view on this let's leave it as-is and I'll just write something in the help text explaining that it defaults to `true` for backwards-compatibility reasons but that you'd probably want to change it to `false`.